vscode/snippets/snippets-gen/pom.xml

```diff
@@ -67,7 +66,7 @@
             <artifactId>maven-plugin-api</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.apache.commons</groupId>
+            <groupId>commons-io</groupId>
             <artifactId>commons-io</artifactId>
             <scope>compile</scope>
         </dependency>
```

Comment:
In addition we need to get rid of the org.apache.commons dependency as well

``` xml 
       <dependency>
                <groupId>org.apache.commons</groupId>
                <artifactId>commons-io</artifactId>
                <version>${commons.io.version}</version>
            </dependency>
```

Reply:
Done